packages/react-api/package.json

```diff
@@ -9,6 +9,7 @@
     "@polkadot/api": "^4.16.2",
     "@polkadot/extension-dapp": "^0.38.7",
     "@polkadot/x-rxjs": "^6.10.1",
+    "@substrate/connect": "0.3.10",
```

Comment:
```suggestion
```

Added it in https://github.com/polkadot-js/apps/pull/5644 (have too many conflicts each time there is a merge...)